WormsApp/worm.cpp

```diff
@@ -14,12 +15,34 @@ using std::unordered_set;
 using std::hash;
 using std::vector;
 
+std::mutex mut;
+
 void Worm::move() {
+
+
   int oldX = x_, oldY = y_;
 
-  // TODO: Implement the body of the function. It should perform one step
-  // based on currDir_ and modify class variables x_ and y_.
+   //TODO: Implement the body of the function. It should perform one step
+   //based on currDir_ and modify class variables x_ and y_.
+
+  switch (currDir_)
+  {
+  case LEFT:
+      (x_>0)?--x_:x_%=board_->getWidth();
```

Comment:
It looks like obfuscated code :) Please consider putting some spaces to make it readable.

Reply:
Done.